packages/core/src/shareback/metadata/get-metadata-xml.ts

```diff
@@ -0,0 +1,61 @@
+import { UPLOADS_FOLDER } from "../../domain/document/upload";
+import { createFolderName } from "../../domain/filename";
+import { S3Utils, executeWithRetriesS3 } from "../../external/aws/s3";
+import { XDSRegistryError } from "../../external/carequality/error";
+import { Config } from "../../util/config";
+import { capture } from "../../util/notifications";
+
+const region = Config.getAWSRegion();
+const s3Utils = new S3Utils(region);
+const bucket = Config.getMedicalDocumentsBucketName();
+
+export async function getMetadataDocumentContents(
```

Comment:
everything is as it was, besides:
- new func name
- new param: `failGracefully`